test/api/workflows_format_2/converter.py

```diff
@@ -1,14 +1,13 @@
-import os
-import sys
+"""Functionality for converting a Format 2 workflow into a standard Galaxy workflow."""
+from __future__ import print_function
 
-import yaml
+from collections import OrderedDict
```

Comment:
pep8 recommends the ordering stdlib, third party libraries, then local application, so it could be argued this should probably be moved to adhere to that barring some reason it can't be.

That said, to @nsoranzo's question, t doesn't explicitly say 'from' comes after direct import that I can find, though I also do follow that usually.